README.md

```diff
@@ -1,9 +1,92 @@
 # Reactive GraphQL
+> GraphQL reactive executor
 
-This project aims to become a complete GraphQL implementation based around [RxJS](https://github.com/ReactiveX/rxjs).
+[![npm version](https://badge.fury.io/js/reactive-graphql.svg)](https://badge.fury.io/js/reactive-graphql)
 
-## Usage Example [![Edit](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/DanielMSchmidt/reactive-graphql-demo/tree/master/?hidenavigation=1)
+Execute GraphQL queries against reactive resolvers (resolvers that return Observable) to get a reactive results.
```

Comment:
Normal resolvers are also supported, the key is that we return a stream, so that you get updates all the time without needing to re-query.